Update WizardLM 13B V1.1 results

victorsungo opened the pull request:
Update WizardLM 13B V1.1 results

rtaori commented:
thanks for the contribution! if the prompt file is the same as https://github.com/tatsu-lab/alpaca_eval/blob/main/src/alpaca_eval/models_configs/wizardlm-13b/prompt.txt, could we update the `configs.yaml` to point directly to it instead of duplicating the same prompt?

victorsungo commented:
> thanks for the contribution! if the prompt file is the same as https://github.com/tatsu-lab/alpaca_eval/blob/main/src/alpaca_eval/models_configs/wizardlm-13b/prompt.txt, could we update the `configs.yaml` to point directly to it instead of duplicating the same prompt?

@rtaori Thanks. I have deleted the wizardlm-13b-v1.1/prompt.txt and updated the configs.yaml to point directly to the wizardlm-13b/prompt.txt
